src/lib.rs

```diff
@@ -162,3 +160,6 @@ pub use service_info::{AsIpAddrs, IntoTxtProperties, ServiceInfo, TxtProperties,
 
 /// A handler to receive messages from [ServiceDaemon]. Re-export from `flume` crate.
 pub use flume::Receiver;
+
+/// Re-export to keep backward compatiable.
```

Comment:
Nit: Is this comment really needed? I guess It should make no difference to this crate consumers as their code will still work after version upgrade and they won't have any need to reference this comment.

Reply:
You're right. I've updated this comment to be more useful.